queries.sql

```diff
@@ -31,3 +31,43 @@ WHERE NAME <> 'Gabumon';
 SELECT *
 FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;  
+
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+
+
+BEGIN;
+DELETE
+FROM animals;
+ROLLBACK;
```

Comment:
- [ ] you should UPDATE the animals table by setting the species column to unspecified, then roll back the change and verify the change was made.